include/swift/Reflection/ReflectionContext.h

```diff
@@ -52,6 +52,25 @@ template <> struct MachOTraits<8> {
   using Section = const struct llvm::MachO::section_64;
   static constexpr size_t MagicNumber = llvm::MachO::MH_MAGIC_64;
 };
+
+template <unsigned PointerSize> struct ELFTraits;
+
+template <> struct ELFTraits<4> {
+  using Header = const struct llvm::ELF::Elf32_Ehdr;
+  using Section = const struct llvm::ELF::Elf32_Shdr;
+  using Offset = llvm::ELF::Elf32_Off;
+  using Size = llvm::ELF::Elf32_Word;
+  static constexpr unsigned char ELFClass = llvm::ELF::ELFCLASS32;
+};
+
+template <> struct ELFTraits<8> {
```

Comment:
Instead of the `8`, can we specialize using `llvm::ELF::ELFCLASS64`?

Reply:
Idem.